Sources/Player/OfflineEngine/Internal/Storage/GRDBStorage/GRDBOfflineStorage.swift

```diff
@@ -122,7 +117,22 @@ private extension GRDBOfflineStorage {
 	static func databaseURL() throws -> URL {
 		let appSupportURL = PlayerWorld.fileManagerClient.applicationSupportDirectory()
 		let directoryURL = appSupportURL.appendingPathComponent("PlayerOfflineDatabase", isDirectory: true)
-		try PlayerWorld.fileManagerClient.createDirectory(at: directoryURL, withIntermediateDirectories: true)
+		if PlayerWorld.fileManagerClient.fileExists(atPath: directoryURL.path, isDirectory: nil) {
+			var attributes = try? PlayerWorld.fileManagerClient.attributesOfItem(directoryURL.path)
```

Comment:
shouldn't we also log an error if this call throws it?

Reply:
yes, we can log it. 
I just don't want it to cause a crash. If it fails, execution should continue normally.